css/styles.css

```diff
@@ -0,0 +1,67 @@
+.cat {
+    width: 25%;
+    margin: 10px 0 10px 0;
+}
+
+.featured-work {
+    color: #ACACAC;
+    font-weight: bold;
+}
+
+.pictures-featured-work {
+    text-align: center;
+}
+
+
+/*Declaring icons color.*/
+.icon-google-plus2 {
+    color: #B61414;
+}
+
+.icon-github {
+    color: #000000;
+}
+
+.icon-facebook2 {
+    color: #3151D0;
+}
+
+
+/*Leave a space between the icon and text.*/
+span {
+    margin-right: 5px;
+    font-size: 150%;
```

Comment:
#### Nitpick :warning: 

I recommend you to use pixels(px) in your text, because if you want to keep integrity your text won't change, if you use percentages(%) when you visualize your page in different devices the text change of size, and the best is that the text adapts to the device. If you have doubts you can see the next link: 

http://kyleschaeffer.com/development/css-font-size-em-vs-px-vs-pt-vs/